.github/workflows/checks.yml

```diff
@@ -1,3 +1,8 @@
+# we’re generally picking macos-10.15 rather than latest
+# because macos-11 is currently much slower
+# however we would prefer macos-latest where possible to
+# make this CI more resilient to time
```

Comment:
FWIW, `macos-latest` still points to `macos-10.15`, and likely will continue to do so for some time after `macos-11` is broadly available.

That's interesting to hear about the performance difference right now, though.

Reply:
Good point. I’ll amend